File: 2026-2/sesion-de-clase-5/dotnet/Lab01Lib/AssessmentItem.cs
using PUCP.Labs.Lab01.Preguntas;

namespace PUCP.Labs.Lab01.Examenes;

public class AssessmentItem
{
    public Question Pregunta { get; }
    public int Puntaje { get; }
    public bool EsCorrecto { get; set; }

    public AssessmentItem(Question pregunta, int puntaje)
    {
        Pregunta = pregunta;
        Puntaje = puntaje;
    }
}

File: 2026-2/sesion-de-clase-03/lab01-2026-1/AssessmentItem.java

public class AssessmentItem {

    private Question pregunta;
    private int puntaje;
    private boolean esCorrecto;

    public AssessmentItem(Question pregunta, int puntaje) {
        this.pregunta = pregunta;
        this.puntaje = puntaje;
    }

    public void setEsCorrecto(boolean esCorrecto) {
        this.esCorrecto = esCorrecto;
    }

    public Question getPregunta() {
        return pregunta;
    }

    public int getPuntaje() {
        return puntaje;
    }

    public boolean getEsCorrecto() {
        return esCorrecto;
    }
}
